### ISSUE TITLE
Wireshark will not come up

### ISSUE THREAD

**Issue by jleman** [imunes/imunes#83]
First, thanks for the great software.  

When running experiments in IMUNES I am unable to get Wireshark to come up.  The option is available when I right click on nodes, but when I click on Wireshark and select the port to monitor, nothing comes up.  Through other means I can tell that the simulation is running and network traffic is being passed as expected, but it would be nice to have the in-program Wireshark call functional.  Any suggestions?

I am running IMUNES version 2.2.2 on FreeBSD 12.0 in a Hyper-V virtual machine.  I also have Wireshark 2.6.5 installed in FreeBSD.

**Comment by idahogray** [comment]
I wanted to respond for Jon since i built the VM he is using. The VM was built by first installing FreeBSD 12.0 in Hyper-V. I then installed all of the dependencies using `pkg`. Finally, I checked out the FreeBSD-12 branch from github and ran `make install` and `imunes -p` per the instructions.

The error I get if I try to run `wireshark` from a terminal is shown below.
![image](https://user-images.githubusercontent.com/1375485/54753571-59d4f780-4baf-11e9-841d-dfdffdbee9c4.png)

**Comment by denisSal** [comment]
Hi!

> I wanted to respond for Jon since i built the VM he is using. The VM was built by first installing FreeBSD 12.0 in Hyper-V. I then installed all of the dependencies using `pkg`. Finally, I checked out the FreeBSD-12 branch from github and ran `make install` and `imunes -p` per the instructions.

Ok, this is how I installed it too...

> The error I get if I try to run `wireshark` from a terminal is shown below.
> ![image](https://user-images.githubusercontent.com/1375485/54753571-59d4f780-4baf-11e9-841d-dfdffdbee9c4.png)

This is a normal error, because IMUNES runs wireshark differently (using `/usr/local/bin/startxcmd`) so it's not possible to run it from the terminal.

It runs normally for me, but I'll try to help you...

First, run an experiment with one PC node named `pc1` for example. Open terminal on your FreeBSD host machine and run: 

``` # startxcmd pc1 wireshark ```

This should open Wireshark window.

Denis

**Comment by idahogray** [comment]
Okay, that makes sense.

When I run the `startxcmd`, I get the following error:

![image](https://user-images.githubusercontent.com/1375485/54756110-87bd3a80-4bb5-11e9-9e95-dc418654a3ae.png)

**Comment by denisSal** [comment]
Couple of questions...

How are logged in to your FreeBSD VM? Locally or over SSH? Which user is running X on the machine? Are you logged in as some user and sudo to root or is there only root?

Look at the `/usr/local/bin/startxcmd` script and try running it one line at a time and checking output of each command to see if something's not right.

**Comment by idahogray** [comment]
I am logged in locally, via the Hyper-V console. The `keith` user is running X. I then run `sudo imunes`.

I tried using `sudo -i` and then running `imunes`. The same error happened.

I then logged out of the display manager, logged in as root, restarted the display manager, ran `imunes` and wireshark worked.

This resolves the issue for me and I will work with @jleman to get it resolved for him.

Thanks for your time and help.
